blocks/single-column-layout-block/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@wpmedia/single-column-layout-block",
+  "version": "0.1.0",
+  "description": "Single Column Layout Block",
+  "homepage": "https://github.com/WPMedia/arc-themes-blocks",
+  "license": "CC-BY-NC-ND-4.0",
+  "main": "index.js",
+  "files": [
+    "chains",
```

Comment:
```suggestion
    "layouts",
```

Reply:
This is probably why it's not working for me. :/